cmd/packages.go

```diff
@@ -132,6 +132,11 @@ func setPackageFlags(flags *pflag.FlagSet) {
 		"include dockerfile for upload to Anchore Enterprise",
 	)
 
+	flags.StringArrayP(
+		"exclude", "", nil,
+		"exclude paths using a glob expression",
```

Comment:
nit: I don't want to make this too verbose, but I think "exclude" can be ambiguous with our tool, and it might be worthwhile to specify that this is excluding paths **from being scanned** (i.e. Syft will never touch them), as opposed to other kinds of exclusion Syft could've been doing, like filtering output.